examples/tutorial/app/simple_wx.py

```diff
@@ -0,0 +1,56 @@
+# -*- coding: utf-8 -*-
```

Comment:
Can you add this line:

https://github.com/vispy/vispy/blob/master/examples/tutorial/app/shared_context.py#L2

It tells Travis not to try running this example as a standard testable example.